packages/client/src/runtime/core/model/aggregates/aggregate.ts

```diff
@@ -0,0 +1,82 @@
+import type { Client } from '../../../getPrismaClient'
+import type { ModelAction } from '../applyModel'
+import { aggregateMap } from './utils/aggregateMap'
+
+/**
+ * Transforms the `userArgs` for the `.aggregate` shorthand. It is an API sugar
+ * for not having to do things like: `{select: {_avg: {select: {age: true}}}}`.
+ * The goal here is to desugar it into something that is understood by the QE.
+ * @param userArgs to transform
+ * @returns
+ */
+export function desugarUserArgs(userArgs: object) {
+  const _userArgs = desugarCountInUserArgs(userArgs)
+  const userArgsEntries = Object.entries(_userArgs)
+
+  return userArgsEntries.reduce(
+    (aggregateArgs, [key, value]) => {
+      if (aggregateMap[key] !== undefined) {
+        // if it's an aggregate, we re-wrap into select
+        aggregateArgs['select'][key] = { select: value }
+      } else {
+        aggregateArgs[key] = value // or leave it alone
+      }
+
+      return aggregateArgs
+    },
+    { select: {} } as object,
+  )
+}
+
+/**
+ * Desugar `userArgs` when it contains `{_count: true}`.
+ * @param userArgs the user input
+ * @returns
+ */
+function desugarCountInUserArgs(userArgs: object) {
+  if (typeof userArgs['_count'] === 'boolean') {
```

Comment:
Is `{ _count: false }` a valid input here?

Reply:
It aligns with what we were doing previously. So not a breaking change but I'm curious of the output.